### DISCUSSION TITLE
Lazer miss don't match stable (with skins)

### DISCUSSION THREAD

**Discussion by Biokax** [ppy/osu#15206] [Q&A] [answered]
Skin : 
--Yugen--
`https://drive.google.com/file/d/1Q3MaHetOeWgkuxp29TM7ZHjGp_-TiWAO/view`

Stable have falling animation on miss but lazer don't, so it feel weird to play

Stable
![stable1](https://user-images.githubusercontent.com/50368878/138138009-0cd1e165-4e41-4986-abab-cb7acd8210ef.gif)

Lazer
![ezgif com-gif-maker](https://user-images.githubusercontent.com/50368878/138177919-ca4d4009-4c01-467c-b2c0-b453cba5e476.gif)

**Comment by peppy** [comment, marked as answer]
I don't think this needs immediate action unless there is a huge outcry against it.
